packages/service/tests/unit/test_discovery_client.py

```diff
@@ -24,6 +24,8 @@
     ServiceLocation as GrpcServiceLocation,
     UnregisterServiceRequest,
     UnregisterServiceResponse,
+    EnumerateServicesRequest,
+    EnumerateServicesResponse,
```

Comment:
nit: These should be placed above the RegisterServiceRequest following the alphabetical order.